radix-engine-interface/src/api/types/re_node.rs

```diff
@@ -144,12 +144,14 @@ pub enum ComponentOffset {
     Info,
     State,
     RoyaltyConfig,
+    RoyaltyAccumulator,
 }
 
 #[derive(Debug, Clone, TypeId, Encode, Decode, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub enum PackageOffset {
-    Package,
+    Info,
     RoyaltyConfig,
+    RoyaltyAccumulator,
```

Comment:
As per discussion, would like to see Royalty `Config` + `Accumulator` be a part of a unified interface/substate-offsets. But for the purpose of getting things integrated asap, this is good enough for now.